.babelrc

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Added babel for parsing ES6 code in all browsers. This will help in converting all files to modules. We don't need to add `type: "modules"` in package.json now.